docs/guide/robot_sbc/setup_raspberry_pi.md

```diff
@@ -219,8 +219,8 @@ git clone https://github.com/autorope/donkeycar
 cd donkeycar
 git checkout master
 pip install -e .[pi]
-pip install tensorflow==1.13.1
 pip install numpy --upgrade
+pip install https://github.com/lhelontra/tensorflow-on-arm/releases/download/v2.2.0/tensorflow-2.2.0-cp37-none-linux_armv7l.whl
```

Comment:
We should not be using this. There are precompiled wheels on Pi Wheels that should be picked up. Google also makes precompiled binaries available. 

Switch to something like `tensorflow==2.3.0`.

Reply:
Let me check this, I couldn't find armv7 on google for tf 2.2.0, but I'll try 2.2.3

Reply:
https://www.piwheels.org/project/tensorflow  

Shows `2.3.1` as the most recent version of TF.

Reply:
Through pypi we only get:
```bash
pip install tensorflow==
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
ERROR: Could not find a version that satisfies the requirement tensorflow== (from versions: 0.11.0, 1.12.0, 1.13.1, 1.14.0)
```
On the TF page (https://www.tensorflow.org/install/pip) google only have a binary for tf==2.3.1 on py3.5 for armv7 which doesn't install on python 3.7: 
```bash
pip install https://storage.googleapis.com/tensorflow/raspberrypi/tensorflow-2.3.0-cp35-none-linux_armv7l.whl
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
ERROR: tensorflow-2.3.0-cp35-none-linux_armv7l.whl is not a supported wheel on this platform.
```
The other option would be to compile it ourselves from https://www.tensorflow.org/install/source_rpi#python-3.7, I can give it a shot.

Reply:
That's because the `tf` version is still set to `1.x` in `setup.py`.

Reply:
There is no `tf` in `setup.py`.

Reply:
Sorry, I sent you a new change: https://github.com/autorope/donkeycar/pull/643